tests/monotouch-test/Foundation/DateTest.cs

```diff
@@ -72,8 +77,50 @@ public void Precision32022 ()
 		{
 			var a = NSDate.Now;
 			var b = a.SecondsSinceReferenceDate - ((NSDate) (DateTime) a).SecondsSinceReferenceDate;
-			// ensure decimals are not truncated - but there's an unavoidable loss of precision
-			Assert.AreEqual (b, 0, 0.00001, "1");
+
+			if (IntPtr.Size == 4) {
+				// ensure decimals are not truncated - but there's an unavoidable loss of precision
+				Assert.AreEqual (b, 0, 0.00001, "1");
+			}
```

Comment:
Another nitpick: the closing brace, `else` keyword and closing brace all go on the same line.